handler/proxy_client.go

```diff
@@ -43,6 +43,7 @@ type ProxyClient struct {
 	HostOverride        string
 	RegionOverride      string
 	LogFailedRequest    bool
+	UnsignedPayload     bool
```

Comment:
Is this variable used, do we need this variable in this struct?

Reply:
Corrected. It would be great if we can merge this PR as it is blocking the launch blog.